support/proxy/vwf.example.com/sceneGetter.vwf.yaml

```diff
@@ -20,8 +20,16 @@ properties:
       if ( !this.scene ) {
         var sp = this.scenePath || "/";
         this.scene = this.find( sp )[ 0 ];
+        // If this is the prototype object then it will think that it is the 
+        //  scene (because it is directly attached to the root) - in that case, 
+        //  we want to make sure that we don't incorrectly set this.scene, so 
+        //  just short-circuit and return undefined.
+        if ( this.scene === this ) {
+          return undefined;
```

Comment:
I think that the way you had it before where it sets `this.scene` back to `undefined` before returning it is important.  Otherwise, though this `get` will properly return `undefined`, the next one will still return the prototype behavior.